docs/competition/matches/shepherding.md

```diff
@@ -19,41 +19,54 @@ Here is a description of the roles related to shepherding.
+More specifically the guard should not allow anyone into the staging area unless they are a blueshirt or staging for the next match. Only one person from each team is allowed in the staging area and they must wear a high-vis vest showing their team name, indicating that they have passed the safety test. When letting a team (=person with a robot) into the staging area the guard should tell them which corner they will stage for, ideally mentioning the colour and approximate location of the staging desk if possible. We recommend that the guard keeps a record of which teams have already been admitted for a particular match. If the guard sees a team leaving the staging area with robot markers still attached to their robot, she should request that the team return the markers before leaving the staging area.
 
 #### Badgers
 
 The responsibility of the badgers is to supervise the staging as well as to hand out and return robot badges and USB-keys.
 
-More specifically, the badgers should make sure that teams follow the direction of travel through the staging area as indicated by the arrows on the floor. When staging, the badgers should help to guide the teams to the correct staging bay. The badgers then hand out the robot badges and match keys to the teams, making sure that a team has staged in the correct slot and is receiving the correct badges. When destaging after a match, the badgers need to make sure that the teams coming from the staging desks into the arena travel the right way round, not obstructing the destaging. The teams leaving the arena should be advised to place their robots on the destaging desks for badge and USB key removal. The badgers need to collect the badges and keys and sort them, ready to be handed out in the next round of staging. The badgers need to ensure that teams do not leave the staging area with markers or USB-keys. The badgers should also clear any items left on the staging tables at this point to prepare for the next round of staging.
+More specifically, the badgers should make sure that teams follow the direction of travel through the staging area. Once the teams have entered staging the badgers then hand out the robot badges and match keys to the teams, making sure that a team has staged in the correct slot and is receiving the correct badges. The teams leaving the arena should be advised to place their robots on the destaging desks for badge and USB key removal. The badgers need to collect the badges and keys and sort them, ready to be handed out in the next round of staging. The badgers need to ensure that teams do not leave the staging area with markers or USB-keys. The badgers should also clear any items left on the staging tables at this point to prepare for the next round of staging.
+
+### Roving Shepherd
```

Comment:
nit: markdown encourages a newline after headings.

(We don't have a style guide yet, but I'd like to introduce one and ideally a formatter too)